crates/optimism/node/src/args.rs

```diff
@@ -30,6 +30,10 @@ pub struct RollupArgs {
     #[arg(long = "rollup.compute-pending-block")]
     pub compute_pending_block: bool,
 
+    /// Enables interop support in the node.
+    #[arg(long = "rollup.interop")]
+    pub interop: bool,
+
```

Comment:
this is usually implemented as function on chainspec "is_interop_active", but this is good for testing purposes, we'll fix it later